src/renderer/src/components/WalletSelect.tsx

```diff
@@ -19,6 +19,10 @@ type TProps = {
   onSelect?: (wallet: IWalletState) => void
   wallets: IWalletState[]
   balanceExchange: UseMultipleBalanceAndExchangeResult
+  dontShowWatches?: boolean
```

Comment:
You don't need this prop, as we are passing the wallets as props, just pass the filtered wallets.